### PR TITLE
React integration

### PR DESCRIPTION
**Pull request by VR333**

### CODE REVIEW

**Comment by rokor** [comment]
Ok lets start from the top. Please show to everyone and after that remove all example, style and public folders. Only integration, configuration and other related to basic config should be present in PR to master.

File: config/webpacker.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  # Additional paths webpack should lookup modules
+  # ['app/assets', 'engine/foo/app/assets']
+  resolved_paths: []
+
+  # Reload manifest.json on all requests so we reload latest compiled packs
+  cache_manifest: false
+
+  extensions:
+    - .coffee
```
Comment (rokor) [inline config/webpacker.yml]: do we need .coffee, .erb, .ts ...?

**Review by rokor — CHANGES_REQUESTED** [review]

**Comment by VR333** [comment]
Do you want me to leave all changes untill tomorrow, so I can show others how it works?

**Comment by rokor** [comment]
@VR333  Create separate branch with example. From this one remove everything not related to basic configuration.

**Comment by VR333** [comment]
Got it

**Comment by rokor** [comment]
@VR333 Also, please add description and update header of the ticket to the appropriate format.

**Comment by VR333** [comment]
Done.

**Review by vlodkopits — DISMISSED** [review]

**Comment by vlodkopits** [comment]
Add header to PR like {Jira-ticket-id}: {ticket-desc}, add description to PR (something like: add react and config it ), make sure that your branch is up-to-date with `develop` branch too, not only with master

**Review by vlodkopits — CHANGES_REQUESTED** [review]
make changes from the previous comment

**Comment by VR333** [comment]
Adding header to PR in format {Jira-ticket-id}: {ticket-desc} is impossible due to absence of this story in Jira.

File: Gemfile [hunk 2]
```diff
@@ -30,4 +33,12 @@ group :development do
   gem 'web-console', '>= 3.3.0'
 end
 
+group :test do
```
Comment (okohuch) [inline Gemfile:36]: I see you add them here as well. Why?????

Comment (okohuch) [reply Gemfile:36]: @vlodkopits

Comment (VR333) [reply Gemfile:36]: No, i don't. This gem is present in develop branch, however my branch wasn't merged with develop. Only >>>>'gem 'webpacker', git: 'https://github.com/rails/webpacker.git'<<< this line of code was added by me.

Comment (okohuch) [reply Gemfile:36]: Do you know what is rebase?

Comment (VR333) [reply Gemfile:36]: Yes, u are right. Changes from develop branch was merged* to this one without, and not rebased. That is the cause and the only reason. Further, i will only use git rebase. Thank you)

**Pull request closed without merging**